### PR TITLE
add isis neighbor parsing

### PR DESCRIPTION
**Pull request by wasabi222**
I've added iosxr messages for isis neighbors going up/down but I can't find the proper openconfig models. my reference for previous parsers is here: http://ops.openconfig.net/branches/master/openconfig-isis.html#

These should pass tests but I wanted to see if there was a better model to put these under before merging.

### CODE REVIEW

File: napalm_logs/config/iosxr/ISIS_NEIGHBOR_DOWN.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  # This may change if we are able to find a more well defined naming system.
+  - error: ISIS_NEIGHBOR_DOWN
+    tag: ROUTING-ISIS-5-ADJCHANGE
+    values:
+      neighbor: ([^ ]+)
+      interface: ([^ ]+)
+      level: (L\d)
+      downReason: (.*)
+    line: 'Adjacency to {neighbor} ({interface}) ({level}) Down, {downReason}'
+    model: NOMODEL
```
Comment (mirceaulinic) [inline napalm_logs/config/iosxr/ISIS_NEIGHBOR_DOWN.yml]: Here the model should point to oc-isis, i.e., `model: openconfig-isis`

File: napalm_logs/config/iosxr/ISIS_NEIGHBOR_DOWN.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    mapping:
+      variables:
+        isis/neighbor: neighbor
```
Comment (mirceaulinic) [inline napalm_logs/config/iosxr/ISIS_NEIGHBOR_DOWN.yml]: The hierarchy defined here under `variables` will reflect the structure of the document generated. The `openconfig-isis` YANG module is derived from the same `openconfig-network-instances`, as well as `openconfig-ospf`. Can you have a look at the similar model here: https://github.com/napalm-automation/napalm-logs/blob/master/napalm_logs/config/junos/OSPF_NEIGHBOR_DOWN.yml#L17?

File: napalm_logs/config/iosxr/ISIS_NEIGHBOR_DOWN.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+messages:
+  # 'error' should be unique and vendor agnostic. Currently we are using the JUNOS syslog message name as the canonical name.
```
Comment (mirceaulinic) [inline napalm_logs/config/iosxr/ISIS_NEIGHBOR_DOWN.yml]: Please remove the first comment line.

**Comment by wasabi222** [comment]
cool I think this should do it.

**Comment by mirceaulinic** [comment]
Thanks @wasabi222

**Pull request merged**